### ISSUE TITLE
Table To Polyline (Pro) test fails test_table_to_polyline

### ISSUE THREAD

**Issue by mfunk** [Esri/military-tools-geoprocessing-toolbox#73]
This one seems to be a problem or error in the model **Table To Polyline** and occurs in the Pro version of the tool and not the Desktop version.

From the error message below it is something in the **Convert Coordinate Notation** that is not being passed correctly. Specifically the notation type is not one of the listed types.

```
...
...
Testing Table To Polyline (Pro).
Running tool (Table To Polyline)
ERROR: Start Time: Thursday, May 19, 2016 3:44:36 PM
Failed to execute. Parameters are not valid.
ERROR 000800: The value is not a member of DD_1 | DD_2 | DD_NUMERIC | DDM_1 | DDM_2 | DMS_1 | DMS_2 | GARS | GEOREF | UTM_BANDS | UTM_ZONES | USNG | MGRS.
ERROR 000800: The value is not a member of DD_1 | DD_2 | DD_NUMERIC | DDM_1 | DDM_2 | DMS_1 | DMS_2 | GARS | GEOREF | UTM_BANDS | UTM_ZONES | USNG | MGRS.
The value is empty.
The value is empty.
Failed to execute (TableToPolyline).
Failed at Thursday, May 19, 2016 3:44:36 PM (Elapsed Time: 0.02 seconds)
Start Time: Thursday, May 19, 2016 3:44:36 PM
Failed to execute. Parameters are not valid.
ERROR 000800: The value is not a member of DD_1 | DD_2 | DD_NUMERIC | DDM_1 | DDM_2 | DMS_1 | DMS_2 | GARS | GEOREF | UTM_BANDS | UTM_ZONES | USNG | MGRS.
ERROR 000800: The value is not a member of DD_1 | DD_2 | DD_NUMERIC | DDM_1 | DDM_2 | DMS_1 | DMS_2 | GARS | GEOREF | UTM_BANDS | UTM_ZONES | USNG | MGRS.
The value is empty.
The value is empty.
Failed to execute (TableToPolyline).
Failed at Thursday, May 19, 2016 3:44:36 PM (Elapsed Time: 0.02 seconds)
...
...
```

**Comment by mfunk** [comment]
DOH!!!!

In the Pro version of the tool the parameters are messed up. The order should be:

1) Input Table
2) Input Coordinate Format
3) X Field (Longitude)
4) Y Field (Latitude)
5) Final Output ---- **NEEDS A MAJOR NAME CHANGE**
6) Line Field
7) Sort Field 

But what I see is:

![image](https://cloud.githubusercontent.com/assets/3093182/15485565/eb9fd9c0-20f4-11e6-9b2e-a42165843957.png)

The problem is that the *Input Coordinate Format* is now the last parameter when it should be the second parameter.

**Comment by mfunk** [comment]
Fixed in #146
